components/base/kinematicbase/execution.go

```diff
@@ -49,11 +49,11 @@ type arcStep struct {
 }
 
 func (step *arcStep) String() string {
-	return fmt.Sprintf("Step: lin velocity: %f, ang velocity: %f, duration: %f s, arcSegment %+v, arc start pose %s",
+	return fmt.Sprintf("Step: lin velocity: %f,\n\t ang velocity: %f,\n\t duration: %f s,\n\t arcSegment %s,\n\t arc start pose %v",
 		step.linVelMMps,
 		step.angVelDegps,
 		step.durationSeconds,
-		step.arcSegment,
+		step.arcSegment.String(),
```

Comment:
I think if you are using the `%s` formatting directive this is implied and therefore unnecessary

Reply:
ooo, I'll double check this on my end and update.

Reply:
i think we need it "fmt.Sprintf format %s has arg step.arcSegment of wrong type go.viam.com/rdk/motionplan/ik.Segment"